Aptoide/aptoide-client-v8
# [DT-709] build initial payment UI

**What does this PR do?**

   This PR aims at implementing the initial payments UI. This currenytly only contains the portrait one as the landscape one is being developed, but it should be quick. We separated just to avoid having other people waiting.

Database changed?

No

Where should the reviewer start?

 MainPaymentsView.kt
How should this be manually tested?

Run the app ricardo developed, send an app link and check it opens ok. Remeber you need to have a release version from GH.

What are the relevant tickets?

Tickets related to this pull-request: [DT-709](https://aptoide.atlassian.net/browse/DT-709)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new keys, etc.) 



**Code Review Checklist**


[DT-709]: https://aptoide.atlassian.net/browse/DT-709?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
